libs/designer-ui/src/lib/dynamicallyaddedparameter/index.tsx

```diff
@@ -22,15 +24,42 @@ export interface DynamicallyAddedParameterProps {
   descriptionPlaceholder?: string;
   renderDescriptionField?: boolean;
   required?: boolean;
+  shouldDisplayAddDropdownOption?: boolean;
```

Comment:
It looks like `shouldDisplayAddDropdownOption` and `shouldDisplayRemoveDropdownOption` are always the same value. Can they be merged?

Reply:
Touched base offline. Improved the readability of code to make it more clear.